test/gui/shared/steps/steps.py

```diff
@@ -1466,3 +1463,71 @@ def step(context, username, foldername):
 @Then("credentials wizard should be visible")
 def step(context):
     waitForObject(AccountConnectionWizard.CREDENTIAL_PAGE)
+
+
+@When("user selects configure_synchronization_manually option in advanced section")
+def step(context):
+    clickButton(
+        waitForObject(
+            names.syncModeGroupBox_configureSyncManuallyRadioButton_QRadioButton
+        )
+    )
+    clickButton(waitForObject(names.setupWizardWindow_nextButton_QPushButton))
+
+
+@Then("dialog add folder sync connection should be visible")
+def step(context):
+    mouseClick(
+        waitForObject(
+            names.add_Folder_Sync_Connection_FolderWizardSourcePage_OCC_FolderWizardLocalPath
+        ),
+        271,
+        92,
+        Qt.NoModifier,
+        Qt.LeftButton,
+    )
+    clickButton(
+        waitForObject(
+            names.add_Folder_Sync_Connection_qt_passive_wizardbutton1_QPushButton
+        )
+    )
+
+
+@Then("dialog select a remote destination folder should be visible")
+def step(context):
+    mouseClick(
+        waitForObject(names.add_Folder_Sync_Connection_groupBox_QGroupBox),
+        125,
+        25,
+        Qt.NoModifier,
+        Qt.LeftButton,
+    )
+    mouseClick(
+        waitForObjectItem(names.groupBox_folderTreeWidget_QTreeWidget, "ownCloud"),
```

Comment:
```suggestion
        waitForObjectItem(names.groupBox_folderTreeWidget_QTreeWidget, "ownCloud"),
```
Why `ownCloud` as second argument? If it is mandatory, use variable to store that value.

Reply:
`"ownCloud"` is the name of an item selected by the picker. I don't think assigning an item name to a variable is better in this case. If you all think so then I will assign it.
![Screenshot from 2022-06-03 13-24-47](https://user-images.githubusercontent.com/66173400/171810191-2c510bb6-a58d-4caa-b117-76efee2d7ef7.png)
?

Reply:
I would have used a variable, if I were you.